commands/init/assembly/__tests__/as-pect-example.spec.ts

```diff
@@ -0,0 +1,42 @@
+class Vec3 {
```

Comment:
do we really need this ? maybe the url of the as-pect documentation is enough

Reply:
this true our boilerplate are not really good as for now

https://github.com/massalabs/massa-sc-toolkit/issues/70